app/models/user.rb

```diff
@@ -208,6 +208,10 @@ def self.upload_user_list(user_class, user_list, encoding)
           conflict_target: [:user_name], columns: [:last_name, :first_name, :section_id, :email, :id_number]
         }
         User.where(id: imported.ids).each do |user|
+          if user_class == Ta
```

Comment:
I'm unsure why this block is necessary

Reply:
When we are doing a bulk import TA list, which is not triggering the callback in the TA model. That callback is used for creating associated grader permissions. So I am triggering that callback explicitly here if it is a TA class.